mitodl/iso-3166-2.js
# Fixing state's name by replacing Risal with Rizal

#### What are the relevant tickets?
fixes https://github.com/mitodl/micromasters/issues/4653

#### What's this PR do?
Fixes typo in a state's name by replacing `Risal` with `Rizal`

#### How to test?
On micromaster do the following steps:
1. Log in and go to /profile/personal
2. Under "Where are you currently living" set the Country to "Philippines" 
3. Open the drop-down for "State or Territory"
4. Search for state "Rizal", you will see "Risal" instead.

Now go to the package.json of micromasters and change `iso-3166-2` dependency's commit id with `hamza/replace-Risal-by-Rizal` (ie: "iso-3166-2": "https://github.com/mitodl/iso-3166-2.js/#hamza/replace-Risal-by-Rizal",)
Now govern the same 4 steps mentioned above, you will see that now "Rizal" is showing up.

OR you can also test it by:
```
iso3166.subdivision("PH", "RIZ")
{
  type: 'Province',
  name: 'Rizal',      # new name
  countryName: 'Philippines',
  countryCode: 'PH',
  code: 'PH-RIZ',
  regionCode: 'RIZ'
}
```

#### Screenshots:
Before:
![image](https://user-images.githubusercontent.com/42243411/95199183-dfea7a80-07f5-11eb-973e-f57cbcbc0f4f.png)
After:
![image](https://user-images.githubusercontent.com/42243411/95197900-bc263500-07f3-11eb-8e7d-f7d66afcb224.png)
